jira-project-avatar-as-favicon.user.js

```diff
@@ -0,0 +1,22 @@
+// ==UserScript==
+// @name         Jira: Project icon as tab icon
+// @namespace    http://tampermonkey.net/
+// @version      1
+// @license      MIT
+// @description  Changes browser tab icon to JIRA project icon
+// @author       Sergey Lukashevich
+// @match        https://jira.example.com/browse/*
+// @icon         https://jira.atlassian.com/favicon.ico
+// @homepageURL  https://github.com/rybak/atlassian-tweaks
+// @grant        none
+// ==/UserScript==
+
+(function() {
+    'use strict';
+
+    let project_avatar = document.getElementById('project-avatar');
```

Comment:
Other userscripts in the repo are using camelCase, not snake_case.

Reply:
There is also a type in script and it doesn't work right now :)